travis/build-wheels.sh

```diff
@@ -28,22 +29,15 @@ ls -al /opt/python/
 PACKAGES="intbitset \
 bitarray \
 simplejson \
-tinyarray \
 pyahocorasick \
-billiard \
-psutil \
 requests \
 pycryptodome \
 lxml \
-url \
-url==0.1.6 \
+urlpy \
 markupsafe \
 scandir \
 pyyaml==3.12 \
-pyyaml \
-pyroaring \
-edlib \
-cdifflib 
```

Comment:
Why do you want to remove these? I am playing with them

Reply:
ok , i am adding back all the package, remove whatever you want. Your call